### PR TITLE
copy pasted previous branch over; there are nav issues?

### PR DESCRIPTION
**Pull request by me-liu**
### Description

[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

- screens and nav for name/email/pword and address
- error handling for password
- frontend

### Screenshots

[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy!"

## How to review

[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
- copy pasted from the other signup page

## Next steps

[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

- the navigation is a little odd, although it looks like it was that way from main? Not sure why though since the nav doesn't seem to have changed. Can't move from the profile -> address in signup and can't see the login page when navigating from the welcome screen. pls help Stephanie ty

### Online sources

[//]: # 'Optional - copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @stephaniewong2

### CODE REVIEW

File: src/app/(Authentication)/SignUp/Address/index.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+import { signUpUser } from '../../../../supabase/queries/auth';
+
+export default function SignUpScreen() {
+  const { name } = useLocalSearchParams() as unknown as { name: string };
+  const { email } = useLocalSearchParams() as unknown as { email: string };
+  const { password } = useLocalSearchParams() as unknown as {
+    password: string;
+  };
+
+  const [streetAddress, setStreetAddress] = useState('');
```
Comment (stephaniewong2) [inline src/app/(Authentication)/SignUp/Address/index.tsx:17]: Type all `useStates`!!

File: src/app/(Authentication)/SignUp/Address/index.tsx [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+          setPlaceholder={setPlaceholderCity}
+        />
+      </View>
+
+      <View style={styles.inputWrap}>
+        <View style={styles.smallInputBox}>
+          <Text style={styles.displayTextSmall}>
+            {displayState ? 'State' : ' '}
+          </Text>
+          <TextInput
```
Comment (stephaniewong2) [inline src/app/(Authentication)/SignUp/Address/index.tsx:116]: Let's use Folasade's `MiniAuthInput` for these

File: src/app/(Authentication)/SignUp/Profile/index.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
+import { router } from 'expo-router';
+import React, { useState } from 'react';
+import { Text, View } from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+
+import styles from './styles';
+import AuthInput from '../../../../Components/AuthInput/AuthInput';
+
+export default function SignUpScreen() {
+  const [name, setName] = useState('');
```
Comment (stephaniewong2) [inline src/app/(Authentication)/SignUp/Profile/index.tsx:10]: Type all `useStates`

**Review by stephaniewong2 — CHANGES_REQUESTED** [review]

**Comment by ronniebeggs** [comment]
Code was copied into a new PR.

**Pull request closed without merging**